.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,23 @@
+name: Feature request - General feature
+description: Request the addition of a new feature/functionality
+labels: ["feature request"]
+body:
+  - type: textarea
+    attributes:
+      label: "Description of the desired feature"
+      description: |
+        Please be as detailed as you can in your description.
+        If possible, include an example of how you would like to use this feature (even better if it's a code example).
+  - type: dropdown
+    id: help
+    attributes:
+      label: Are you willing to help implement and maintain this feature?
+      description: |
+        Every feature we add is code that we will have to maintain and keep updated. This takes a lot of effort.
+        If you are willing to be involved in the project and help maintain your feature, it will make it easier for us to accept it.
+      options:
+        - "No (Default)"
```

The '(Default)' text would show up. Let's just do No/Yes/Maybe.
```suggestion
        - "No"
```